src/ProcessMaker/Nayra/Contracts/Engine/JobManagerInterface.php

```diff
@@ -0,0 +1,57 @@
+<?php
```

Comment:
This interface defines the different types of schedules that a timer event could use.

Reply:
Wouldn't it be better to add this git comment as part of the interface comment?

Reply:
Yes